_data/toc.yaml

```diff
@@ -1,5 +1,5 @@
 toc:
-- title: Welcome to the Docs
+- title: What is Docker?
```

Comment:
No entry is viable in the case of the home page; either way, the left-nav tree will be completely collapsed. The only difference is there wouldn't be a blue-colored nav entry on the left side.